Cu 3075b6m create a real footer with navigation

BenDanielGreep opened the pull request:
## Objective
What: Redesign the footer
Why: It was a image before
How: Using @matthew1809's footer from CT
Usage: Main.jsx

## Type

- [ ] Bug Fix
- [ ] New Feature
- [ ] Performance Tweaks
- [X] Style Tweaks
- [X] Code Refactoring
- [ ] Documentation
- [ ] Tests


## Tested

Locally, will test on netlify


## Documented

- [ ] Have you documented in changed file using comments
- [ ] Have you added instructions to the README if needed?

diegolarraz commented:
Task linked: [CU-3075b6m Create a real footer with Navigation](https://app.clickup.com/t/3075b6m)

matthew1809 commented:
<img width="1512" alt="Screenshot 2022-10-05 at 17 22 33" src="https://user-images.githubusercontent.com/10894967/194098655-3b549387-d57f-4795-bc61-d8e9f53d93f6.png">

Would suggest a top border as seen here, and a tad more spacing to make it feel more comfortable. 

Know I didn't do this, but centring the trademark statement similar to picture wouldn't hurt either.

Those are personal points of view, the code is great!

matthew1809 commented on src/Main.jsx:
```diff
@@ -137,8 +137,10 @@ export const Main = () => {
+          {/* NB disabled logic to render footer */}
           {/* To avoid CLS, load in the footer after the carousels render */}
-          {carouselLoaded && <Footer />}
+          {/* {carouselLoaded && <Footer />} */}
```
Why is this left in at all?

BenDanielGreep replied:
Left in because I would eventually like to re-add this functionality
I should've commented

matthew1809 reviewed (commented):
GG @BenDanielGreep!

HugoWith reviewed (changes requested):
Hey Ben, 

First of all thanks for handling that, it's clean 👏

Few comments if you have the time, if you don't please let me know I can help with that 👌

#1 On mobile view : if possible align to the top or to the center the three blocks

![Screenshot 2022-10-05 at 19 04 39](https://user-images.githubusercontent.com/47173348/194120411-eb28a37f-21c0-464f-aa0d-cca925dd7a89.png)

#2 Small responsive issue, everything is cluttered

![Screenshot 2022-10-05 at 19 04 18](https://user-images.githubusercontent.com/47173348/194120508-97654ef7-4f98-419c-a861-ca49e73426e2.png)

#3 Align to top or stretch the first block to balance the left side of the footer

![Screenshot 2022-10-05 at 19 02 04](https://user-images.githubusercontent.com/47173348/194120628-9a2e544a-9b8a-4728-9698-0c3df8b4f957.png)

#4 It would be super nice to have a real navigation on this footer. We could use the navigation element from the navbar, we could also open the cart for examples. Among fake button of course 😊

#5 We already talk about this earlier but the html and the CSS are a bit complexe for what it does (I know it's legacy code from something made with bootsrap). If you have the time and the energy it would be great to simplify everything and use BEM synthaxe, using rem, % and not only px. But totally understand if you can't, we have a perfectly functional footer already so well done 😊

BenDanielGreep commented:
@HugoWith implemented the styling  changes requested.
Regarding the legacy code - I've added BEM where I adjusted but I think it would be better to put the full rebuild into another ticket. Happy to take it when we have the bandwidth.

BenDanielGreep commented:
@HugoWith yes I would include them both in a single ticket, do you agree?
